src/js/edu-benefits/5495/components/IntroductionPage.jsx

```diff
@@ -60,26 +60,34 @@ class IntroductionPage extends React.Component {
             </li>
           </ol>
         </div>
-        <div className="row progress-box progress-box-schemaform form-progress-buttons schemaform-buttons">
-          <div className="small-6 usa-width-five-twelfths medium-5 columns">
-            <a href="/education/apply-for-education-benefits/">
-              <button className="usa-button-outline">« Back</button>
-            </a>
-          </div>
-          <div className="small-6 usa-width-five-twelfths medium-5 end columns">
-            <ProgressButton
-              onButtonClick={this.goForward}
-              buttonText="Continue"
-              buttonClass="usa-button-primary"
-              afterText="»"/>
-          </div>
-        </div>
-        <div className="omb-info--container">
+        <SaveInProgressIntro
+          pageList={this.props.route.pageList}
+          messages={this.props.route.formConfig.savedFormMessages}
+          {...this.props.saveInProgressActions}
+          {...this.props.saveInProgress}>
+          Please complete the 22-5495 form to apply for education benefits.
```

Comment:
It looks like we have inconsistent language here and in the 1990e, do we need to confirm the content for this across these forms?

Reply:
Yeah, we probably should. I think this one can at least be changed to match the format in HCA/1990e/pension/burials. I think that language is from Peggy. We can create an issue for her to go back through them and confirm.

Reply:
Yeah, we probably should. I don't think it needs to hold this up, but we need to have her look at them.